examples/subplots_axes_and_figures/figure_size_units.py

```diff
@@ -0,0 +1,78 @@
+"""
+==============================
+Figure size in different units
+==============================
+
+The native figure size unit in matplotlib is inches, deriving from print
+industry standards. However, users may need to specify their figures in other
+units like cm or pixels. This example illustrates how to do this efficiently.
+"""
+
+# sphinx_gallery_thumbnail_number = 2
+
+import matplotlib.pyplot as plt
+text_kwargs = dict(ha='center', va='center', fontsize=28, color='C1')
+
+##############################################################################
+# Figure size in inches (default)
+# -------------------------------
+#
+plt.subplots(figsize=(6, 2))
+plt.text(0.5, 0.5, '6 inches x 2 inches', **text_kwargs)
+plt.show()
+
+
+#############################################################################
+# Figure size in cm
+# -----------------
+# Multiplying cm-based numbers with a conversion factor from cm to inches,
+# gives the right numbers. Naming the conversion factor ``cm`` makes the
+# conversion almost look like appending a unit to the number, which is nicely
+# readable.
+#
+cm = 1/2.54  # cm in inches
```

Comment:
I was 90% sure this was exact, but had to look it up to be 100% sure.  You may want to indicate that with a link to https://en.wikipedia.org/wiki/Inch? and/or more decimal places?  Or at least say its exact?

Reply:
IMHO that'd be overly cautious. One should assume that code/logic is correct unless explicitly indicated as an approximation. I'd really feel awkward adding something like "Look, what we're doing is really correct. Here is the proof".

Reply:
OK, but doing `1/2.5400` wouldn't hurt?